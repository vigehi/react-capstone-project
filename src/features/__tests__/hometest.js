/* eslint-disable no-unused-vars */
import { Provider } from 'react-redux';
import { render, screen } from '@testing-library/react';
import Home from '../home';
import store from '../../app/store';

describe('Test the post', () => {
  it('renders correctly', () => {
    const post = [
      {
        post: 'Tata Power, a top power producer in India, confirms cyberattack',
        info: '<p>Tata Power, a leading power generation company in India, has confirmed it was hit by a cyberattack. In a brief statement released on Friday, the Mumbai-based company said that the attack impacted some of its I.T. systems. &#8220;The company has taken steps to retrieve and restore the systems. All critical operational systems are functioning; however, [&hellip;]</p>',
      },
    ];

    const { container } = render(
      <Provider store={store}>
        <Home />
      </Provider>,
    );
    const message = screen.getByText('Enter a post');
    expect(message).toMatchSnapshot();
  });
});
