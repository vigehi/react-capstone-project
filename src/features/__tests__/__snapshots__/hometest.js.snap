// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Post list renders correctly 1`] = `
<div
  class="heading"
>
  Enter a valid post!
</div>
`;
